pkg/operator/operator.go

```diff
@@ -405,6 +405,34 @@ func New(
 	}
 	o.controllersToRunFunc = append(o.controllersToRunFunc, csrController.Run, o.ruleController.Run, o.relabelController.Run)
 
+	csrFederateController, err := csr.NewClientCertificateController(
+		csr.ClientCertOption{
+			SecretNamespace: "openshift-monitoring",
+			SecretName:      "metrics-client-certs",
```

Comment:
you want a different secret name I believe.

Reply:
Right, sorry copy paste